.github/workflows/pypi-publish.yml

```diff
@@ -25,11 +25,23 @@ jobs:
     permissions:
       id-token: "write"
     steps:
+      - name: Get publish-requirements.txt from repository
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+        with:
+          sparse-checkout: |
+            .github/workflows/requirements/publish-requirements.txt
+          sparse-checkout-cone-mode: false
+          # The tag to build or the tag received by the tag event
+          ref: ${{ github.event.inputs.version || github.ref }}
```

Comment:
There's no version variable in this publish workflow, so this is always going to be github.ref right?